test/e2e/networking.go

```diff
@@ -141,7 +175,7 @@ var _ = Describe("Networking", func() {
 			}
 		}()
 
-		By("Wait for the webserver pods to be ready")
+		By("Waiting for the webserver pods to transition to Running state")
```

Comment:
since its a test, i chose to keep it in this block - and use that function as a utility instead... is that ok?